scripts/calculate_contributors.py

```diff
@@ -0,0 +1,127 @@
+            print(f"Error fetching repos: {r.status_code} {r.text}")
+            break
+        data = r.json()
+        if not data:
+            break
+        repos += [repo["name"] for repo in data]
+        page += 1
+    return repos
+
+def get_commits(repo, start_date, end_date):
+    """
+    Fetch commits for a repository within the date range.
+    """
+    commits = []
+    page = 1
+    headers = {"Authorization": f"token {os.getenv('GITHUB_TOKEN')}"}
+    
+    while True:
+        url = (f"https://api.github.com/repos/{ORG}/{repo}/commits?"
+               f"since={start_date.isoformat()}Z&until={end_date.isoformat()}Z&per_page=100&page={page}")
+        r = requests.get(url, headers=headers)
+        if r.status_code != 200:
+            print(f"Error fetching commits for {repo}: {r.status_code}")
+            break
+        data = r.json()
+        if not isinstance(data, list) or not data:
+            break
+        commits += data
+        page += 1
+    return commits
+
+def main():
+    """
+    Calculate top contributors and format the Community Spotlight message.
+    """
+    # Initialize contributor tracking.
+    contribution_count = defaultdict(int)
+    
+    # Get date range and repositories.
+    start_date, end_date = get_date_range()
+    repos = get_repos()
+
+    # Count commits per author.
+    for repo in repos:
+        commits = get_commits(repo, start_date, end_date)
+        for commit in commits:
+            author = commit.get("author")
+            if author and author.get("login"):
+                contribution_count[author["login"]] += 1
+
+    # Sort contributors by commit count.
+    top_contributors = sorted(contribution_count.items(), key=lambda x: x[1], reverse=True)
+
+    # Build the message.
+    message = (
+        "👥 **Community Spotlight Reminder** 🙌\n\n"
+        "Here are the top GitHub contributors to all **Scribe** projects from "
+        f"**{start_date.strftime('%B %d')}** to **{end_date.strftime('%B %d')}**:\n\n"
+    )
+    for user, count in top_contributors[:TOP_N]:
+        message += f"- [{user}](https://github.com/{user}) ({count} commits)\n"
+
+    message += (
+        "\n💬 Please reply in the thread with who you think should be spotlighted this month!\n"
+        "We'll reach out to them for a short message and LinkedIn tag before posting. 💙\n"
+    )
+
+    # Write message to file for reference.
+    with open("message.txt", "w") as f:
+        f.write(message)
+
+    # Set GitHub Actions output.
+    with open(os.getenv("GITHUB_OUTPUT"), "a") as f:
+        f.write(f"message<<EOF\n{message}\nEOF\n")
+
+    # Log summary for debugging.
+    with open(os.getenv("GITHUB_STEP_SUMMARY"), "a") as f:
+        f.write("**Community Spotlight Summary** 🟢\n")
+        f.write(f"Date Range: {start_date.strftime('%Y-%m-%d')} to {end_date.strftime('%Y-%m-%d')}\n")
+        f.write(f"Repositories Scanned: {len(repos)}\n")
+        f.write("Top Contributors:\n")
```

Comment:
It is really an excellent idea bought by @angrezichatterbox ✨ and work done by @axif0 👏🏾 

I just think it would be better to exclude Scribe maintainers from the spotlight list. As it will highly motivate new comers to contribute more if they see their name in the list. 
However, it is just my opinion.

Reply:
Ya I'd agree that organization members don't need to be included :) But then your idea of people seeing their own name does raise the question of whether this is going to `Operations` for us to decide and share on LinkedIn, or if this goes to `General` for everyone to see and then we'd post about one person on LinkedIn. 

Thoughts on this? :)

Reply:
@axif0, @mhmohona, @angrezichatterbox: What are we thinking about having this be automated into the General channel vs. to Operations? I'm really fine either way, but let's remember that if it's public and then we decide to "highlight" someone on social media that we'd need to make sure that we communicate well with the community so there's no hard feelings for one person being singled out. I think we can for sure do this well though 😊 Maybe reaching out to someone about the community spotlight post is via a personal message, and we can discuss the results of this workflow in Operations to see who we should reach out to?

Once we decide on this, I'd be happy to add in the needed secrets and then we can launch this 🚀

Reply:
> Maybe reaching out to someone about the community spotlight post is via a personal message

Yes, its really an important thing to do. With this, we should ask their social media handle so it can be ensured that we are tagging the right person.

Reply:
I think we can just do a conversation about this in Operations and have the monthly message be a reminder that we could do a community spotlight post on social media as well 😊 How does this sound?